format/FormatNexusEigerDLS16M.py

```diff
@@ -60,6 +69,11 @@ def __init__(self, image_file, **kwargs):
 
         super(FormatNexusEigerDLS16M, self).__init__(image_file, **kwargs)
         self._dynamic_shadowing = self.has_dynamic_shadowing(**kwargs)
+        try:
+            meta = image_file.replace("_master.h5", "_meta.h5")
```

Comment:
This won't work if the .nxs file is provided as input.

Reply:
Added equivalent code for `.nxs`